server/configs/application.properties

```diff
@@ -141,16 +142,16 @@ management.server.port=@@shutdownPort@@
 
 ## START OF CSP REPORT BLOCK (DO NOT CHANGE THIS TEXT)
 csp.report=\
-    default-src 'self' ;\
-    connect-src 'self' ${LABKEY.ALLOWED.CONNECTIONS} ;\
-    object-src 'none' ;\
-    style-src 'self' 'unsafe-inline' ;\
-    img-src 'self' data: ;\
-    font-src 'self' data: ;\
-    script-src 'unsafe-eval' 'strict-dynamic' 'nonce-${REQUEST.SCRIPT.NONCE}' ;\
-    base-uri 'self' ;\
-    frame-ancestors 'self' ;\
-    report-uri /admin-contentsecuritypolicyreport.api?${CSP.REPORT.PARAMS} ;
+    default-src 'self' ;                                                            /* Default sourcing locations for resources if explicit declarations not present */\
+    connect-src 'self' ${LABKEY.ALLOWED.CONNECTIONS} ;                              /* Allowed connection sources, can be substituted and appended via the LabKey Admin UI */\
+    object-src 'none' ;                                                             /* Allowed sources for object, embed, and applet tags */\
+    style-src 'self' 'unsafe-inline' ;                                              /* Allowed sources for style tags */\
+    img-src 'self' data: ;                                                          /* Allowed sources for imgages */\
+    font-src 'self' data: ;                                                         /* Allowed font sources */\
+    script-src 'unsafe-eval' 'strict-dynamic' 'nonce-${REQUEST.SCRIPT.NONCE}' ;     /* Allowed sources for script tags */\
+    base-uri 'self' ;                                                               /* Allowed sources for base tags */\
+    frame-ancestors 'self' ;                                                        /* Allowed sources for embeded and other framed resourcees */\
```

Comment:
I think this comment is describing `frame-src`. I think `frame-ancestors` is for who is allowed to embed labkey in a frame on a different site.

Reply:
https://content-security-policy.com/frame-ancestors/ is where I pulled the comment from

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors seems to fit with Trey's interpretation